config/default/manager/manager.yaml

```diff
@@ -188,3 +198,13 @@ webhooks:
         apiGroups: ["kubevirt.io"]
         apiVersions: ["v1alpha3"]
         resources: ["virtualmachines"]
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: service
+  namespace: system
+type: kubernetes.io/tls
+data:
+  tls.crt: YmFkIGNlcnRpZmljYXRlCg==
```

Comment:
What do these values stand for?

Reply:
```bash
$ echo "YmFkIGNlcnRpZmljYXRlCg==" | base64 -d
bad certificate
```

Reply:
We need this in order to triger the manager to create new cert?